website/pages/api-docs/agent/service.mdx

```diff
@@ -513,13 +513,14 @@ service definition keys for compatibility with the config file format.
 
 - `Kind` `(string: "")` - The kind of service. Defaults to "" which is a
   typical Consul service. This value may also be "connect-proxy" for
-  services that are [Connect-capable](/docs/connect)
-  proxies representing another service or "mesh-gateway" for instances of
-  a [mesh gateway](/docs/connect/mesh_gateway)
+  [Connect-capable](/docs/connect) proxies representing another service,
+  "mesh-gateway" for instances of a [mesh gateway](/docs/connect/mesh_gateway),
+  "terminating-gateway" for instances of a [terminating gateway](/docs/connect/terminating_gateway),
+  or "ingress-gateway" for instances of a [ingress gateway](/docs/connect/ingress_gateway).
 
 - `Proxy` `(Proxy: nil)` - From 1.2.3 on, specifies the configuration for a
-  Connect proxy instance. This is only valid if `Kind == "connect-proxy"` or
-  `Kind == "mesh-gateway"`. See the [Proxy documentation](/docs/connect/registration/service-registration)
+  Connect proxy instance. This is only valid if `Kind != ""`.
```

Comment:
```suggestion
  Connect service proxy instance. This is only valid if `Kind` is configured.
```

Reply:
This isn't quite right. The empty kind means "typical service". 

How about:
```suggestion
  Connect service proxy instance. This is only valid if `Kind` defines a proxy or gateway.
```